src/ConnectedMode/ConnectedMode.csproj

```diff
@@ -34,8 +34,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Remove="Embedded\SonarLintTargets.xml" />
-    <EmbeddedResource Include="Embedded\SonarLintTargets.xml" />
+    <None Remove="Install\SonarLintTargets.xml" />
+    <EmbeddedResource Include="Install\SonarLintTargets.xml" />
```

Comment:
I moved the resource file so all of the "install" related files are in the same folder.